checks/basic/unused_secrets.go

```diff
@@ -126,7 +126,7 @@ func checkReferences(objects *kube.Objects) (map[kube.Identifier]struct{}, error
 	return used, g.Wait()
 }
 
-// envVarsSecretRefs checks for config map references in container environment variables
+// envVarsSecretRefs checks for secret references in container environment variables
 func envVarsSecretRefs(containers []corev1.Container, namespace string) []kube.Identifier {
```

Comment:
Looks like there's another bug where https://github.com/digitalocean/clusterlint/blob/master/checks/basic/unused_secrets.go#L115 should use `envVarsSecretRefs`. Can you please update this too and add a test for the same?

Reply:
Hi @varshavaradarajan,

I've added two more tests that are copies of the tests for non-init containers. The functions I've added for creating the test objects are copies too just with the change to make the pods have InitContainers rather than Containers. There might be a clever way to create the objects without the repeated code, but I've not explored that as I didn't want to add any complication to your tests.